spec/plugins/aws-lambda/99-access_spec.lua

```diff
@@ -730,6 +749,17 @@ for _, strategy in helpers.each_strategy() do
       assert.equal(65, tonumber(res.headers["Content-Length"]))
     end)
 
+    it("errors on bad region name (DNS resolution)", function()
+      local res = assert(proxy_client:send {
+        method  = "GET",
+        path    = "/get?key1=some_value1",
+        headers = {
+          ["Host"] = "lambda15.com"
+        }
+      })
+      assert.res_status(500, res)
```

Comment:
Could we also ensure that the error logs will contain a clear message indicating what the issue is? There are some new helper utilities in the Kong test suite to verify the error logs as such.

Reply:
Not sure those helpers already made it into EE version CI runs against

Reply:
Easy to cherry-pick then.

Reply:
not sure  I understand? we cannot cherry-pick it into older releases?

See https://github.com/Kong/kong-plugin-aws-lambda/blob/master/.travis.yml#L5-L16

Reply:
@thibaultcha @Tieske I've added an assertion without using the new helpers to ensure compatibility, and also updated the readme to describe the expected behavior if an invalid region name is provided.